controllers/NUgus_Controller/tcp.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef CONTROLLERS_NUGUS_CONTROLLER_TCP
+#define CONTROLLERS_NUGUS_CONTROLLER_TCP
+
+#include <sys/socket.h>
+#include <vector>
+
```

Comment:
Are we going to put this in namespace `Controllers::NUgus_Controller`?

Reply:
That namespace makes me uncomfortable, what is it trying to be? camel case? snake case? a weird mix of the two?

Reply:
I think we should aim for `controllers::nugus`